rdr_service/dao/metrics_cache_dao.py

```diff
@@ -1471,10 +1478,11 @@ def get_metrics_cache_sql(self, hpo_id):
                            ELSE 0
                          END AS Multi_Ancestry,
                          CASE
-                           WHEN (PMI_Skip = 1 AND Number_of_Answer=1) OR UNSET = 1
+                           WHEN (PMI_Skip = 1 AND Number_of_Answer=1) OR (HasBasicsSurvey = 1 AND UNSET = 1)
                              THEN 1
                            ELSE 0
-                         END AS No_Ancestry_Checked
+                         END AS No_Ancestry_Checked,
+                         CASE WHEN HasBasicsSurvey = 0 AND UNSET = 1 THEN 1 ELSE 0 END AS Unset_No_Basics
```

Comment:
If `HasBasicsSurvey = 0`, UNSET will always be 1. Maybe we can just use  `HasBasicsSurvey = 0`

Reply:
Good point, I've made all these requested changes in the new commit.